### PR TITLE
update contact form - PLEASE REVIEW, MERGE IF GOOD

### PR DESCRIPTION
**Pull request by gordonleacock**
add interactive improvements to contact form

PLEASE REVIEW BUT DON'T MERGE YET

Simple-form isn't generating the aria-required tags I think we'd like. I'll check with Colin or Jon E. tomorrow.

### CODE REVIEW

File: app/views/contacts/_help_sidebar.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div id="sidebar" class="<%= sidebar_classes %>">
```
Comment (respinos) [inline app/views/contacts/_help_sidebar.html.erb]: You can pull in the partial from the static pages, to avoid duplicating the partial.

**Comment by gordonleacock** [comment]
Okay, I'm going to leave the ARIA to later testing.

The current version DRYs out the code by keeping the sidebar code in static only.

**Comment by gordonleacock** [comment]
Ready to merge, check it out.

**For a later sprint:**
1. We can test email when we get the test server running and loaded.
2. Add autofocus for first form field (name) if UX team okays.
3. Roger: I cleaned up application.rb but need to you explain any changes I need to make for xml import.

**Comment by respinos** [comment]
re: xml import --- shouldn't need any changes; your importer doesn't actually look for files/images/etc.

re: the form --- I don't understand why the form isn't being arranged with the bootstrap grid/forms, but that's another iteration. Bit concerned that the custom styles are overriding bootstrap's names and how that'll affect other appearances in the app.

**Pull request merged**